packages/rest/src/router/metadata.ts

```diff
@@ -305,16 +331,13 @@ export function param(paramSpec: ParameterObject) {
       '@param decorator can be applied to methods only',
     );
 
-    let endpoints = Object.assign(
-      {},
-      Reflector.getMetadata(ENDPOINTS_KEY, target),
-    );
-    Reflector.defineMetadata(ENDPOINTS_KEY, endpoints, target);
-
-    let endpoint: Partial<RestEndpoint> = endpoints[propertyKey];
-    if (!endpoint) {
+    const endpoints = getEndpoints(target);
```

Comment:
ditto: should we change ts-docs for this function to say that child classes can override REST parameters?